app/packages/core/src/components/app/App.tsx

```diff
@@ -114,6 +117,8 @@ export const App: React.FunctionComponent<IAppProps> = ({ icons, plugins }: IApp
                         <Layout>
                           <Routes>
                             <Route path="/" element={<Home />} />
+                            <Route path="/plugins" element={<Plugins />} />
+                            <Route path="/plugins/cluster/:cluster/type/:type/name/:id" element={<Plugin />} />
```

Comment:
I would stick with the existing url:

```suggestion
                            <Route path="/plugins/:cluster/:type/:name" element={<Plugin />} />
```

Reply:
This will make the routing more difficult. Right now we can use the `instance.ID` in our `<Link to={instance.id}/>` component to route to the Plugin page. Can we also change the template for the ID parameter, so that we don't have to build the path in the frontend?